viite-UI/src/model/ProjectCollection.js

```diff
@@ -644,23 +657,27 @@
     };
 
 
-    var addToDirtyRoadPartList = function (queryResult) {
+    var addToReservedPartList = function (queryResult) {
       var qRoadParts = [];
-      _.each(queryResult.roadparts, function (row) {
+      _.each(queryResult.reservedInfo, function (row) {
         qRoadParts.push(row);
       });
 
-      var sameElements = arrayIntersection(qRoadParts, currentReservedParts, function (arrayarow, arraybrow) {
+      var sameElements = arrayIntersection(qRoadParts, reservedParts, function (arrayarow, arraybrow) {
         return arrayarow.roadNumber === arraybrow.roadNumber && arrayarow.roadPartNumber === arraybrow.roadPartNumber;
       });
       _.each(sameElements, function (row) {
         _.remove(qRoadParts, row);
       });
       _.each(qRoadParts, function (row) {
-        currentReservedParts.push(row);
+        reservedParts.push(row);
       });
     };
 
+    var addToFormedPartList = function (queryResult) {
```

Comment:
Empty function?

Reply:
Good catch. This can be removed since we dont need it in the Avaa button: "roadPartsValidation:checkRoadParts"